actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -677,7 +677,7 @@ def parse(name, encrypted_message, purpose: nil)
           deserialize(name) do |rotate|
```

Comment:
_(grabbing line of code for convo mode)_
> I think you need a changelog entry

Yup, happy to do that, I didn't because the [contributing guide at `5.5`](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code) says:
> If the change adds, removes, or changes a feature, be sure to include a CHANGELOG entry. If your change is a bug fix, a CHANGELOG entry is not necessary.

But looking at it again, at `5.10` it says:
> You should add an entry to the top of the CHANGELOG of the framework you modified if you're adding or removing a feature, committing a bug fix, or adding deprecation notices.

So I'll do that.

Reply:
Thanks for pointing out the discrepancy! I opened a PR to fix that: #45963

I think this change isn't quite a bug fix (it's more of a change in behavior), which is why I think its important to document it.

Reply:
Good call 👍